src/ert/_c_wrappers/enkf/config_content_transmutor.py

```diff
@@ -0,0 +1,59 @@
+from typing import Any, Dict, List
+
+from ert._c_wrappers.config import ConfigContent
+from ert._c_wrappers.enkf.config_keys import ConfigKeys
+
+SINGLE_VALUE_KEYS = [
+    ConfigKeys.ALPHA_KEY,
+    ConfigKeys.ANALYSIS_SELECT,
+    ConfigKeys.CONFIG_DIRECTORY,
+    ConfigKeys.DATAROOT,
+    ConfigKeys.DATA_FILE,
+    ConfigKeys.ECLBASE,
+    ConfigKeys.ENSPATH,
+    ConfigKeys.GEN_KW_EXPORT_NAME,
+    ConfigKeys.GEN_KW_TAG_FORMAT,
+    ConfigKeys.GRID,
+    ConfigKeys.HISTORY_SOURCE,
+    ConfigKeys.ITER_CASE,
+    ConfigKeys.ITER_COUNT,
+    ConfigKeys.ITER_RETRY_COUNT,
+    ConfigKeys.JOBNAME,
+    ConfigKeys.JOB_SCRIPT,
+    ConfigKeys.LICENSE_PATH,
+    ConfigKeys.MAX_RESAMPLE,
+    ConfigKeys.MAX_RUNTIME,
+    ConfigKeys.MAX_SUBMIT,
+    ConfigKeys.MIN_REALIZATIONS,
+    ConfigKeys.NUM_CPU,
+    ConfigKeys.NUM_REALIZATIONS,
+    ConfigKeys.OBS_CONFIG,
+    ConfigKeys.QUEUE_SYSTEM,
+    ConfigKeys.RANDOM_SEED,
+    ConfigKeys.REFCASE,
+    ConfigKeys.RERUN_KEY,
+    ConfigKeys.RERUN_START_KEY,
+    ConfigKeys.RUNPATH,
+    ConfigKeys.RUNPATH_FILE,
+    ConfigKeys.STD_CUTOFF_KEY,
+    ConfigKeys.STOP_LONG_RUNNING,
+    ConfigKeys.TIME_MAP,
+    ConfigKeys.UPDATE_LOG_PATH,
+]
+
+
+class ConfigContentTransmutor:
+    @staticmethod
+    def as_dict(config_content: ConfigContent) -> Dict[str, List[Any]]:
```

Comment:
In current trunk, there is a new version of `config_content_as_dict` in a separate file. I think that covers your use-case.

Reply:
Yep, it is a near identical implementation. Worked fine! :+1: